exercises/practice/high-scores/high-scores_spec.lua

```diff
@@ -0,0 +1,115 @@
+local high_scores = require 'high-scores'
```

Comment:
```suggestion
local HighScores = require 'high-scores'
```

Since this is a "constructor" it should use `CapitalCamelCase` instead of `snake_case`